model/task/task.go

```diff
@@ -592,6 +596,57 @@ func (t *Task) MarkGeneratedTasks() error {
 	)
 }
 
+func GenerateNotRun() ([]Task, error) {
+	const maxGenerateTimeAgo = 2 * time.Hour
+	return FindAll(db.Query(bson.M{
+		StatusKey:         evergreen.TaskStarted,                              // task is running
```

Comment:
Once an amboy job has generated tasks from this task, it will set the `GenerateTasksKey` of this task to true. This will cause POSTing to tasks/<task_id>/generate to noop, and GETing to tasks/<task_id>/generate to return `true` and any errors that the generator encountered, so you should be able to restart the task any arbitrary amount of times. See the changes beginning https://github.com/evergreen-ci/evergreen/pull/2630/files#diff-a775b07b340fe3d74270110291af05a0R24.